### PR TITLE
Adding PHP translation bundle

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

I have two questions:
* Check my paths for `dir` and `output_dir`, are they correct according to sf4 directory structure? 
* Am I correct by putting "dev only" config in a subfolder called "dev" next to the config shared on all environments?

### CODE REVIEW

File: php-translation/symfony-bundle/0.4/config/packages/dev/php_translation.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+translation:
+    symfony_profiler: # must be placed in config_dev.yml
```
Comment (fabpot) [inline php-translation/symfony-bundle/0.4/config/packages/dev/php_translation.yaml]: Using `packages/dev` is correct 👍

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+translation:
+    locales: ["en", "sv", "fr"]
```
Comment (fabpot) [inline php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: I would only use `["en"]` here.

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    webui:
+        enabled: true
+    edit_in_place:
+        enabled: false
+        config_name: app
+        activator: php_translation.edit_in_place.activator
+    configs:
+        app:
+            dirs: ["%kernel.project_dir%/templates", "%kernel.root_dir%/src"]
+            output_dir: "%kernel.project_dir%/app/Resources/translations"
```
Comment (fabpot) [inline php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: This one is not correct, should be `%kernel.project_dir%/translations`

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+            excluded_names: ["*TestCase.php", "*Test.php"]
+            excluded_dirs: [cache, data, logs]
```
Comment (fabpot) [inline php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: What does it refer to? `var/`? Why?

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+translation:
+    locales: ["en", "sv", "fr"]
+    webui:
+        enabled: true
+    edit_in_place:
+        enabled: false
+        config_name: app
+        activator: php_translation.edit_in_place.activator
```
Comment (javiereguiluz) [inline php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: Could we remove this `activator` option and rely on a default value? It's super redundant that the value of the `activator` option for `edit_in_place` option of PGP translation is `php_translation.edit_in_place.activator`.

**Comment by fabpot** [comment]
@Nyholm Any news on this one?

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml (hunk 4 above)
Comment (Nyholm) [reply php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: These values will be given to `Finder::notPath()`. Ie, when extracting the `src` or `vendor` for translations we should never go in to folders named "cache", "data" or "logs".

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+translation:
+    locales: ["en"]
+    webui:
+        enabled: true
```
Comment (stof) [inline php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: shouldn't this be enabled only in dev, as you import the routing only in dev ?

File: php-translation/symfony-bundle/0.4/config/packages/dev/php_translation.yaml (hunk 1 above)
Comment (stof) [reply php-translation/symfony-bundle/0.4/config/packages/dev/php_translation.yaml]: I suggest removing this `must be placed in config_dev.yml` comment. It is more confusing than anything else here

File: php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml (hunk 6 above)
Comment (Nyholm) [reply php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml]: You are correct. Thank you

**Comment by Nyholm** [comment]
Thank you for the reviews

**Pull request merged**